datadog_checks_base/datadog_checks/base/checks/win/wmi/__init__.py

```diff
@@ -226,48 +264,70 @@ def _submit_metrics(self, metrics, metric_name_and_type_by_property):
             func(metric_name, metric.value, metric.tags)
 
     def _get_instance_key(self, host, namespace, wmi_class, other=None):
+        # type: (str, str, str, Any) -> str
         """
         Return an index key for a given instance. Useful for caching.
         """
         if other:
             return "{host}:{namespace}:{wmi_class}-{other}".format(
                 host=host, namespace=namespace, wmi_class=wmi_class, other=other
             )
-
         return "{host}:{namespace}:{wmi_class}".format(host=host, namespace=namespace, wmi_class=wmi_class)
 
+    def get_running_wmi_sampler(self, properties, filters, **kwargs):
+        # type: (Iterable[str], List[Dict[str, WMIFilter]], Dict) -> WMISampler
```

Comment:
```suggestion
        # type: (Iterable[str], List[Dict[str, WMIFilter]], Any) -> WMISampler
```

Reply:
Python 2 cheasheet suggests:
```suggestion
        # type: (Iterable[str], List[Dict[str, WMIFilter]], **Any) -> WMISampler
```